README.md

```diff
@@ -16,9 +16,9 @@ That's it! `foo` can be a `@rest` query, it could be `@dbquery`, it could be a f
 2. If you do not have `stepzen` installed, then `npm install -g stepzen`.
 3. `cd graphql && stepzen service start && stepzen login --config ~/.stepzen/stepzen-config.local.yaml`--this starts a local docker container running all of StepZen's services.
 4. `stepzen deploy` deploys the `graphql` schema in `index.graphql` to StepZen (on your local machine) and the endpoint is `http://localhost:9000/api/subscription/__graphql`
-5. Now we will have a client that makes subscription requests on `web sockets` to this endpoint. For this, `cd ../client && npm install graphql-ws && npm install ws && npm install dotenv`.
+5. Now we will have a client that makes subscription requests on `web sockets` to this endpoint. For this, `cd ../client && npm install`.
 6. Setup .env; `echo 'APIKEY='$(stepzen whoami --apikey) > .env`  (`stepzen whoami --apikey` retrieves your admin key, it shoud look something like: `graphql::local.net...`)
-7. `node local.js` and see the magic. You will see the price of `ETHBTC` change and reflected back on your console.
+7. `node client.js` and see the magic. You will see the price of `ETHBTC` change and reflected back on your console.
 
 ## How this all works
 The subscription is against a REST backend: `https://api2.binance.com/api/v3/ticker?symbol=ETHBTC` and StepZen does a long polling and pushes back to the client whenever a new value is found. 
```

Comment:
later it has `Because any type Query can be converted to type Subscription`

makes no sense.

Maybe

Because any `Query` field can be exposed as a `Subscription` field ...